bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -491,10 +505,8 @@ impl SlidingSyncView {
     ) -> Arc<StoppableSpawn> {
         let mut room_list = self.inner.rooms_list.signal_vec_cloned().to_stream();
         Arc::new(StoppableSpawn::with_handle(RUNTIME.spawn(async move {
-            loop {
```

Comment:
grrr... leftover from a bad refactor that wasn't fully reverted: we still need this `loop {` here ...caused stupid bug...